modules/apps/dynamic-data-mapping/dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js

```diff
@@ -133,6 +180,42 @@ AUI.add(
 						instance.evaluate();
 					},
 
+					_createField: function(dataType) {
+						var instance = this;
+
+						var parameterMessage = '';
+
+						var selectedValidation = instance.get('selectedValidation');
+
+						if (selectedValidation) {
+							parameterMessage = selectedValidation.parameterMessage;
+						}
+
+						var field;
```

Comment:
Hey @alinedoleron Can you please move the `field` variable closer to it usages?

Thx